frontend/integration-tests-cypress/cypress-e2e/cypress/support/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// in cypress/support/index.d.ts
+// load type definitions that come with Cypress module
+/// <reference types="cypress/types" />
+declare namespace Cypress {
+  interface Chainable {
```

Comment:
Will we always add new helpers to this?

Reply:
It is part of the Cypress documentation to do this in this file.

Reply:
Update: removed `...support/index.d.ts` and moved this code to `index.ts`